terraform-lambda-ecr-integration/templates/lambda-role.json

```diff
@@ -0,0 +1,62 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "BasicDescribePolicy",
+      "Effect": "Allow",
+      "Action": [
+        "cloudwatch:DescribeAlarms",
+        "cloudwatch:PutMetricAlarm",
```

Comment:
Can you please expand the need for DescribeAlarms and PutMetricAlarm permissions here?

Reply:
I added it to give flexibility to add alarms in the cloud watch but for this particular use case it is not needed. I will remove it and push the code again.